lib/ReactTags.js

```diff
@@ -56,6 +56,42 @@ class ReactTags extends React.Component {
     this.setState({ query })
   }
 
+  handlePaste (e) {
+    // allow over-ride, if there is a need
+    if (this.props.onPaste) {
+      return this.props.onPaste(e)
+    }
+
+    const { delimiterChars } = this.props
+
+    e.preventDefault()
+
+    // get the text data from the clipboard
+    const data = e.clipboardData.getData('Text')
+
+    if (data && delimiterChars.length > 0) {
+      // split the string based on the delimiterChars as a regex
+      const tags = data.split(new RegExp(delimiterChars.join('|'), 'g'))
+      for (let i = 0; i < tags.length; i++) {
+        // the logic here is similar to handleKeyDown, but subtly different,
+        // due to the context of the operation
+        if (tags[i].length > 0) {
+          // look to see if the tag is already known
+          const matchIdx = this.suggestions.state.options.findIndex((suggestion) => {
```

Comment:
I has been using the same logic as handleKeyDown, so we should probably have that updated too eventually?